test.nix

```diff
@@ -30,8 +30,12 @@ let
         else
           input
       );
-
+    components = {
+      servisen = c.mkService { name = "plate"; version = "1.2.1"; src = null; };
+    };
```

Comment:
What is this testing?

Reply:
It's now required in order to run the tests since languages.nix requires that as an argument. Added a mock service inside it just to have something in there.

Reply:
Cool, was just that I didn't see it accessed or asserted so it wasn't clear what it was doing there :)